src/Psalm/Internal/Analyzer/StatementsAnalyzer.php

```diff
@@ -145,14 +145,15 @@ public function analyze(
                     && isset($stmt->expr->args[1])
                     && $stmt->expr->args[0]->value instanceof PhpParser\Node\Scalar\String_
```

Comment:
this condition can now be removed I think? If we're using the string value below (though this is rarely used, so it won't make much of a difference)

Reply:
You're correct. I missed this one.